.github/workflows/release-please.yml

```diff
@@ -0,0 +1,65 @@
+name: Run Release Release
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Now that I have switched this to main the workflow will fail until this PR is merged.

Reply:
(Or not run.)